pages/lot/[id].js

```diff
@@ -103,39 +103,27 @@ function OptionalRow({ label, data }) {
   );
 }
 
-const dateFieldOrder = {
-  processed_date: 1,
-  prescreen_completed_date: 2,
-  under_review_date: 3,
-  review_started_including: 4,
-  review_started_date_excluding: 5,
-  collect_permit_fees_date: 6,
-  pending_issuance_date: 7,
-  issue_permit_date: 8,
-  final_date: 9,
-};
-
-const dateFieldLabel = {
-  processed_date: 'Processed',
-  prescreen_completed_date: 'Prescreen',
-  under_review_date: 'Under review',
-  review_started_including: 'Review started',
-  review_started_date_excluding: 'Review started',
-  collect_permit_fees_date: 'Collect fees',
-  pending_issuance_date: 'Pending issuance',
-  issue_permit_date: 'Issued',
-  final_date: 'Done',
+const DATE_FIELDS = {
+  processed_date: { order: 1, label: 'Processed' },
+  prescreen_completed_date: { order: 2, label: 'Prescreen' },
+  under_review_date: { order: 3, label: 'Under review' },
+  review_started_including: { order: 4, label: 'Review started' },
+  review_started_date_excluding: { order: 4, label: 'Review started' },
```

Comment:
to "group" fields as if they represent the same thing, just use the same order on them and that will make sure that only one of them will show up on the list

Reply:
nice